steps/generate-text-file/0.0.1/step.yml

```diff
@@ -0,0 +1,45 @@
+title: Generate Text File
+summary: Dynamically generate a text file
+description: |-
+  This step lets you generate a dynamic text file which can use any environment
+  variables. The path of the file will be made available as
+  `$GENERATED_TEXT_FILE_PATH`. This script can e.g. be used to generate the Play
+  Store's "What's New" file based on the Git commit message.
+website: https://github.com/nicolas-fricke/bitrise-step-generate-text-file
+source_code_url: https://github.com/nicolas-fricke/bitrise-step-generate-text-file
+support_url: https://github.com/nicolas-fricke/bitrise-step-generate-text-file/issues
+published_at: 2017-02-10T13:14:04.386721078+01:00
+source:
+  git: https://github.com/nicolas-fricke/bitrise-step-generate-text-file.git
+  commit: 1dd81d6acaf45388a446f85427cb89abdfb64726
+host_os_tags:
+- osx-10.10
+project_type_tags:
```

Comment:
You should remove all the project type tags, as this step should work with any kind of project (right?)